### ISSUE TITLE
An approver needs to approve a PR after accept2ship label is attached for it to work

### ISSUE THREAD

**Issue by huydhn** [pytorch/test-infra#502]
Here is an example PR https://github.com/pytorch/pytorch/pull/82753. Here is what happened:

1. PR approved
1. The label accept2ship was added, nothing happened
1. PR approved again
1. Now it worked (https://github.com/pytorch/pytorch/pull/82753#issuecomment-1205632998)

I think that 4 should always run as long as the PR is approved and an accept2ship label exists regardless of their order of appearances. Same goes for accept2run label. This is more consistent with people expectation. Thoughts?

cc @janeyx99 @zengk95

**Comment by janeyx99** [comment]
To give context on the OG accept2ship. Internally, accept2ship is often used when the PR author is giving permission to the approver to land the PR whenever it's been accepted (meaning there are no other dependencies to wait on). I think landing on the first approve is a plus from the OG, but I think if the PR is approved and the user puts accept2ship, we don't necessarily have to trigger a land.

The user can just put the ship command "@pytorchbot merge" normally in that case. 

This may be a cool feature, but I'm not sure it's inconsistent with expectations.

**Comment by huydhn** [comment]
I see. I have never used the internal accept2ship before, thus not aware of that original meaning of the label. That's why it feels strange to me that a PR with both accept2ship and approvals doesn't start the merge automatically. If this is expected, let it be I guess unless more people notice this. I haven't seen many people use this label so far, so it's hard to say.

**Comment by janeyx99** [comment]
Yes, I'd be curious about others' expectations too. We can bring it up in our next external and internal developers' syncs and clarify this point.

**Comment by clee2000** [comment]
accept2ship function was removed, closing
